src/components/macro/macro-action-editor/tab/macro-mouse/macro-mouse.component.ts

```diff
@@ -0,0 +1,102 @@
+import {
+    Component, 
+    OnInit, 
+    Input, 
+    Output, 
+    EventEmitter, 
+    ElementRef, 
+    ViewChild
+} from '@angular/core';
+import { FORM_DIRECTIVES } from '@angular/forms';
+import {Tab} from '../../../../popover/tab/tab';
+import {MacroAction, macroActionType} from '../../../../../../config-serializer/config-items/MacroAction';
+
+import { PressMouseButtonsMacroAction } from '../../../../../../config-serializer/config-items/PressMouseButtonsMacroAction';
+import { HoldMouseButtonsMacroAction } from '../../../../../../config-serializer/config-items/HoldMouseButtonsMacroAction';
+import { ReleaseMouseButtonsMacroAction } from '../../../../../../config-serializer/config-items/ReleaseMouseButtonsMacroAction';
+
+enum TabName {
+    Move,
+    Scroll,
+    Click,
+    Hold,
+    Release
+}
+
+@Component({
+    moduleId: module.id,
+    selector: 'macro-mouse-tab',
+    template: require('./macro-mouse.component.html'),
+    styles: [
+        require('../../macro-action-editor.component.scss'),
+        require('./macro-mouse.component.scss')
+    ],
+    host: { 'class': 'macro__mouse' },
+    directives: [ FORM_DIRECTIVES ]
```

Comment:
I think you can remove this line, since `FormsModule` will be imported in app.module.ts